docs/get-started/going-live.md

```diff
@@ -67,6 +67,27 @@ yarn webiny deploy api --env=dev
 
 Since this is a new instance of the `api` stack, it will take some time to deploy. Wait for the stack deployment to finish, before moving to the next step.
 
+### 2.1 Deploy specific `api` resource
+
+From the root of your project, run the following command:
+
+```
+yarn webiny deploy api <resource-name> --env=dev
+```
+
+for example, running this command will only deploy `securityGraphQL` resource
```

Comment:
For example, running this command will only deploy `securityGraphQL` resource:

(Make sure to capitilize the first word and if you are linking to an image or a piece of code, just end with ":" instead of a period or nothing